### PR TITLE
henrik simple jest test

### LINKED ISSUE #88: Write a unit test
## Acceptance Criteria
* There is a new test that improves our code coverage

## References
https://lmiller1990.github.io/vue-testing-handbook/

### PR DESCRIPTION
**Pull request by te4-henrik-stahl**
## What issue did you implement or fix?
Fixed #88

## Summary
- Fixed foobar in zoobar.
- Made foo better.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## How can this implementation be improved?
- Change bar into zar. :shipit:

### CODE REVIEW

**Review by TE4-David-Sundqvist — APPROVED** [review]
👍  test test test

File: lib/tests/unit/Timeline.spec.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+import Vue from "vue";
+import vuetify from "vuetify";
+
+
+it("Check if sign in is in the login page", () => {
+    Vue.use(vuetify);
+  const wrapper = shallowMount(Login)
+  const msg = "Sign in"
+
```
Comment (te4-adrian-almetun-smeds) [inline lib/tests/unit/Timeline.spec.ts:12]: What's up with the indentation?

**Review by te4-adrian-almetun-smeds — CHANGES_REQUESTED** [review]
One small comment

**Review by te4-adrian-almetun-smeds — APPROVED** [review]
I like :+1: Merge it :shipit:

**Pull request merged**